src/dotnet-mssql/.devcontainer/mssql/postCreateCommand.sh

```diff
@@ -58,7 +58,7 @@ then
         then
             dbname=$(basename $f ".dacpac")
             echo "Deploying dacpac $f"
-            /opt/sqlpackage/sqlpackage /Action:Publish /SourceFile:$f /TargetServerName:localhost /TargetDatabaseName:$dbname /TargetUser:sa /TargetPassword:$SApassword
+            /opt/sqlpackage/sqlpackage /Action:Publish /SourceFile:$f /TargetTrustServerCertificate:True /TargetServerName:db /TargetDatabaseName:$dbname /TargetUser:sa /TargetPassword:$SApassword
```

Comment:
@bamurtaugh Do you know of someone with expertise who can help review this PR? 

The change to `/TargetServerName:db` makes sense to me. However, want to double check on `/TargetTrustServerCertificate:True` to ensure any security implications.

Reply:
I wonder if @croblesm might have thoughts, given his expertise in SQL?

Reply:
TrustServerCertificate must be used since the docker image of mssql exposes a self signed certificate which can't be trusted by default. 

I do not think it should be a security concern as the environment is isolated and the app container has access only to the db instance networking-wise.

Reply:
The explanation makes sense, thanks @mattiascibien